docs/src/main/sphinx/connector/kudu.rst

```diff
@@ -11,7 +11,7 @@ Requirements
 
 To connect to Kudu, you need:
 
-* Kudu version 1.10 or higher.
```

Comment:
In " Upgrade kudu client to 1.15.0 " please wrap the commit message body to 72 chars. See https://chris.beams.io/posts/git-commit/ for project's convention.

Reply:
👍 
Any tips for helping format commit messages? Without a linter/some automated tool in place it is something I always forget.

Reply:
Nevermind, was able to configure vim to auto-wrap at 72 chars for only git commit messages. 

~/.vimrc
```
"enable filetype detection
filetype plugin indent on

"auto wrap commit messages at 72 chars
autocmd FileType gitcommit set textwidth=72
autocmd FileType gitcommit set colorcolumn=73
```